src/retrace/retrace.py

```diff
@@ -18,8 +18,9 @@
 from subprocess import *
 import magic
 import six
+from dnf.subject import Subject
+from  hawkey import FORM_NEVRA
```

Comment:
isn't hawkey obsoleted?

Reply:
Yes, hawkey as a project is obsolete, but it is part of libdnf.